build.ps1

```diff
@@ -0,0 +1,46 @@
+﻿$SENTENCES = 5
```

Comment:
We might as well write one single `build.py` script.  It would safe us the trouble of having separate Makefile PowerShell scripts.

Another alternative would be to use `mingw32-make`, for example, from https://sourceforge.net/projects/mingw-w64/files/External%20binary%20packages%20%28Win64%20hosted%29/make/ , avoiding the need for the whole cygwin